.github/workflows/check-test.yaml

```diff
@@ -31,12 +31,16 @@ jobs:
     steps:
       - name: Sets env vars
         if: github.event_name == 'pull_request'
-        run: echo "SINCE_REF=remotes/origin/main" >> $GITHUB_ENV
+        run: |
+          echo "SINCE_REF=remotes/origin/main" >> $GITHUB_ENV # linux/mac
+          echo "SINCE_REF=remotes/origin/main" >> $env:GITHUB_ENV # windows
```

Comment:
Not sure if i should split it based on the OS. Let me know what you think?

Reply:
probably that would be cleaner

Reply:
added in [97a6654](https://github.com/mongodb-js/devtools-shared/pull/355/commits/97a665491af8f922a9310ecee869cc57608e3c32)